packages/components/src/organisms/Filter/Filter.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+import { Accordion as UIAccordion } from '../../'
```

Comment:
Inside the components package, we don't need to use this alias pattern. We are just using on core :)